CHANGELOG.md

```diff
@@ -3,9 +3,13 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 
-## v1.8.27 - TBD
+## v2.0.0 - TBD
 [placeholder for next release]
 
+## v1.8.27 - 2024-08-12
+Update Copyrights to Include 2024
```

Comment:
I see that the changelog for this project sometimes mentions copyright year updates, which surprises me. I don't think it's necessarily helpful to know that the copyright headers in the source files were updated. We should undo the changes to this file.